lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
not sure if its worth the complexity, i mean by default we scope per account, this add some specific scoping by rental, (which is only used in los endpoint for performance reasons), not sure if the generic solution will ever be needed, + i don't really like this solution either